local_db/data_access.py

```diff
@@ -322,6 +322,19 @@ def reject_file(
 
             self._create_audit_log(audit)
 
+    def reset_review_file(
+        self, request_id: str, relpath: UrlPath, username: str, audit: AuditEvent
+    ):
+        with transaction.atomic():
+            request_file = RequestFileMetadata.objects.get(
+                request_id=request_id, relpath=relpath
+            )
+
+            review = FileReview.objects.get(file=request_file, reviewer=username)
```

Comment:
Should this catch the exception if the review doesn't exist? (It's reasonably easy to trigger if you have 2 airlock tabs open )

Reply:
yep, have added handler & tests for this.